app/views/contexts/thank_you_context.py

```diff
@@ -31,6 +31,7 @@ def build_census_thank_you_context(
 ) -> Mapping:
 
     context = {
+        "submitted_time": session_data.submitted_time,
```

Comment:
This needs to be added to the context of all post submission pages.

Reply:
As my comment above, do we want to do that? should it not live in template_helper